src/others/sysmon/lv_sysmon.h

```diff
@@ -77,20 +78,37 @@ typedef struct {
 /**********************
  * GLOBAL PROTOTYPES
  **********************/
+#if LV_USE_SYSMON
+
+#if LV_USE_PERF_MONITOR
+
+/**
+ * Show system performance monitor: CPU usage and FPS count
+ */
+void lv_sysmon_show_performance(void);
+
+/**
+ * Hide system performance monitor
+ */
+void lv_sysmon_hide_performance(void);
+
+#endif /*LV_USE_PERF_MONITOR*/
+
+#if LV_USE_MEM_MONITOR
 
 /**
- * Create a system monitor object.
- * @param parent pointer to an object, it will be the parent of the new system monitor
- * @return       pointer to the new system monitor object
+ * Show system memory monitor: used memory and the memory fragmentation
  */
-lv_obj_t * lv_sysmon_create(lv_obj_t * parent);
```

Comment:
It is recommended to retain `lv_sysmon_create` so that users can implement their own monitors.

Reply:
Is it a real use case?

Reply:
Yes, it is necessary if we have multiple heaps or multiple resources that need to be dynamically monitored.

Reply:
In this case we might be better converting it widget-like by adding a class definition.